next.config.js

```diff
@@ -3,4 +3,6 @@ const nextConfig = {
   reactStrictMode: true,
 }
 
+require('dotenv').config()
```

Comment:
Is there a particular reason we're using the dotenv module? I think next supports loading envs from a .env file [out of the box](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables). You might know something I don't

Reply:
I'm also potentially concerned that we'll expose certain env variables to the browser that we don't want to. If we stick with dotenv, this might warrant further investigation.

Reply:
You're right I didn't realize we could just do it through next

Reply:
Removing it!